src/Workspaces/Core/Portable/Rename/Renamer.RenameDocumentAction.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Rename
+{
+    public static partial class Renamer
+    {
+        /// <summary>
+        /// Individual action from RenameDocument apis in <see cref="Renamer"/>.
+        /// 
+        /// See <see cref="Renamer.RenameDocumentFoldersAsync(Document, System.Collections.Generic.IReadOnlyList{string}, Options.OptionSet, System.Threading.CancellationToken)" />
+        /// and <see cref="Renamer.RenameDocumentNameAsync(Document, string, Options.OptionSet, System.Threading.CancellationToken)" />
+        /// </summary>
+        public abstract class RenameDocumentAction
+        {
+            public ImmutableArray<string> Errors { get; }
```

Comment:
Does this also need GetErrors that takes a CultureInfo like you did for GetDescription?

Reply:
I think so